patches/api/0005-Adventure.patch

```diff
@@ -487,7 +488,21 @@ index c15861905d7af679c4f6ffc27719a900b0fe4284..d5fdc57ffcfac4eb18d7fbf44ce13257
      public static int broadcastMessage(@NotNull String message) {
          return server.broadcastMessage(message);
      }
-@@ -1051,6 +1053,19 @@ public final class Bukkit {
+@@ -820,6 +822,13 @@ public final class Bukkit {
+         return server.getLogger();
+     }
+ 
++    // Paper start - Adventure component logger
++    @NotNull
++    public net.kyori.adventure.text.logger.slf4j.ComponentLogger getComponentLogger() {
```

Comment:
A global logger was a mistake in the first place, don't add another one with component support to extend the mistake